### ISSUE TITLE
Is it possible integrate amazing stuff with my Node+React web app?

### ISSUE THREAD

**Issue by battlebrain** [microsoft/visual-chatgpt#146]
Hi everyone!
Is it possible integrate amazing stuff with my Node+React web app?

**Comment by K-tang-mkv** [comment]
Yeah, it totally can be done. I got the same idea wanna integrate this project into my python+Nextjs web app. Let's do it

**Comment by battlebrain** [comment]
Perfect! Let's keep in touch,
Can I have your contact info?

**Comment by K-tang-mkv** [comment]
For sure, I will create a repo about it, and also you can add my WeChat: `I_Change_World` or follow my Twitter [https://twitter.com/Tang13220820](https://twitter.com/Tang13220820) to message to me

**Comment by battlebrain** [comment]
Perfect, Can we continue our conversation on skype?
My skype name is live:.cid.30df9e27b4ee909b

**Comment by K-tang-mkv** [comment]
OK, I will add you Skype

**Comment by aebubebell** [comment]
can I join in?

**Comment by battlebrain** [comment]
Yes, join! you can add me on skype.

**Comment by albertoecf** [comment]
Good Luck!! ❤️
